createRelease.sh

```diff
@@ -96,19 +96,35 @@ function zip_repo {
     echo "ZIPPING REPO: $REPO_NAME"
     echo $SEPARATOR
     
-    cd $COURSE_RELEASE_FOLDER
+    cd $RELEASE_DIR
     
-    zip -r "$STARTING_DIR/$COURSE_RELEASE_FILE" *
+    zip -r "$STARTING_DIR/$COURSE_RELEASE_FILE" $REPO_NAME
     
     cd $STARTING_DIR
 }
 
+function clean_target {
```

Comment:
This logic should be part of the "prepare_repo" function. It's intention is to run necessary steps to prep the repo for zipping.